### PR TITLE
A bunch of minor translations

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

This includes some minor translations from the following PRs:

- https://github.com/moonbeam-foundation/moonbeam-docs/pull/693/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/677/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/675/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/666/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/655/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/660/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/646/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/647/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/699/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/696/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/651/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/649/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/702/files / https://github.com/moonbeam-foundation/moonbeam-docs/pull/706/files
- https://github.com/moonbeam-foundation/moonbeam-docs/pull/652/files (only `wormhole.md` changes)

### CODE REVIEW

File: builders/build/historical-updates.md [hunk 1]
```diff
@@ -196,6 +196,62 @@ EIP-2612和以太坊区块以秒为单位处理时间戳，然而Moonbeam采用
 
 ***
 
+#### Ethereum Transations Duplicated in Storage {: #ethereum-transactions-duplicated-in-storage }
```
Comment (eshaben) [inline builders/build/historical-updates.md:199]: Just waiting on the title to be translated

File: builders/build/historical-updates.md [hunk 2]
```diff
@@ -118,7 +118,23 @@ EIP-2612和以太坊区块以秒为单位处理时间戳，然而Moonbeam采用
 
 ***
 
-#### 错误委托奖励计算 {: #incorrect-delegation-reward-calculation } 
+#### Substrate Tips Missing Treasury Distribution {: #substrate-tips }
```
Comment (eshaben) [inline builders/build/historical-updates.md:121]: Title to be translated

File: builders/build/historical-updates.md [hunk 3]
```diff
@@ -212,13 +286,27 @@ EIP-2612和以太坊区块以秒为单位处理时间戳，然而Moonbeam采用
 
 ***
 
-## 迁移 {: #migrations } 
+#### Remote EVM Calls Return Identical Transaction Hashes {: #remote-evm-calls-return-identical-tx-hashes }
```
Comment (eshaben) [inline builders/build/historical-updates.md:289]: Title to be translated

File: builders/build/historical-updates.md (hunk 2 above)
Comment (eshaben) [inline builders/build/historical-updates.md:121]:
```suggestion
#### Substrate Tips缺少财政库分配 {: #substrate-tips }
```

File: builders/build/historical-updates.md [hunk 4]
```diff
@@ -196,6 +212,64 @@ EIP-2612和以太坊区块以秒为单位处理时间戳，然而Moonbeam采用
 
 ***
 
+#### Ethereum Transations Duplicated in Storage {: #ethereum-transactions-duplicated-in-storage }
```
Comment (eshaben) [inline builders/build/historical-updates.md:215]:
```suggestion
#### 以太坊交易在存储中重复 {: #ethereum-transactions-duplicated-in-storage }
```

File: builders/build/historical-updates.md (hunk 3 above)
Comment (eshaben) [inline builders/build/historical-updates.md:289]:
```suggestion
#### 远程EVM调用返回相同的交易哈希 {: #remote-evm-calls-return-identical-tx-hashes }
```

File: builders/get-started/endpoints.md [hunk 5]
```diff
@@ -27,13 +27,14 @@ description: 使用支持的API提供商连接至公共端点或创建私有JSON
...
 --8<-- 'text/endpoints/moonbase.md'
 
-## 端点提供商 {: #endpoint-providers } 
+## RPC端点提供商 {: #endpoint-providers }
 
 您可以使用以下任意API提供商，创建适用于开发环境或生产环境的端点：
 
 - [1RPC](#1rpc)
 - [Blast](#blast)
 - [BlockSpaces](#blockspaces)
```
Comment (themacexpert) [inline builders/get-started/endpoints.md:36]: At quick glance, Blockspaces doesn't seem to provide Web3 RPC services anymore. If that's the case we should remove them from the CN and english sites

**Review by themacexpert — APPROVED** [review]
LGTM 👍

File: builders/get-started/endpoints.md (hunk 5 above)
Comment (eshaben) [reply builders/get-started/endpoints.md:36]: oh yeah good catch!! they deprecated it as of 6/1/2023. I'll remove it here and open a PR to remove it on the EN repo

**Pull request merged**